src/main/java/org/springframework/data/elasticsearch/core/mapping/SimpleElasticsearchPersistentProperty.java

```diff
@@ -66,6 +71,15 @@ public SimpleElasticsearchPersistentProperty(Property property,
 
 		this.annotatedFieldName = getAnnotatedFieldName();
 		this.isId = super.isIdProperty() || SUPPORTED_ID_PROPERTY_NAMES.contains(getFieldName());
+
+		// @deprecated since 4.1
+		@Deprecated
```

Comment:
No need fort inline annotations.

Reply:
removed the `@` this comment is just a documentation as hint for later that we know since when we deprecated it